test/test_podman_pods.sh

```diff
@@ -114,4 +114,4 @@ podman pod kill foobar
 ########
 # Remove all pods and their containers
 ########
-podman pod rm -fa
+podman pod rm -t 0 -fa
```

Comment:
Maybe I missed it, but I don't recall seeing a -t value for anything other than zero.  It would be nice to have at least a small handful with values other than zero.

Reply:
Added some.